icaruscode/TPC/SignalProcessing/RecoWire/ROIFinder_module.cc

```diff
@@ -555,9 +588,15 @@ void  ROIFinder::processPlane(size_t                      idx,
                     holder.resize(newSize);
                 }
 
+                // Now make the short int version
+                icarus_signal_processing::VectorShort intHolder(holder.size());
+
+                for(size_t binIdx = 0; binIdx < holder.size(); binIdx++) intHolder[binIdx] = std::round(holder[binIdx]);
```

Comment:
I thought about this but don't know how it can happen. Remember the RawDigits are 12 bit max ranging 0 to 4096. The deconvolution process does not multiply this by factors, generally staying within this range. The waveforms are pedestal subtracted so can have negative values but can't go to the limits of a short int. FLW...

Reply:
Thank you for checking!